nncf/torch/graph/operator_metatypes.py

```diff
@@ -20,68 +22,67 @@
 from nncf.common.graph.definitions import NNCFGraphNodeType
 from nncf.common.graph.layer_attributes import BaseLayerAttributes
 from nncf.common.graph.layer_attributes import ConvolutionLayerAttributes
+from nncf.common.graph.operator_metatypes import UNKNOWN_METATYPES
 from nncf.common.graph.operator_metatypes import INPUT_NOOP_METATYPES
 from nncf.common.graph.operator_metatypes import NOOP_METATYPES
 from nncf.common.graph.operator_metatypes import OUTPUT_NOOP_METATYPES
 from nncf.common.graph.operator_metatypes import OperatorMetatype
 from nncf.common.graph.operator_metatypes import OperatorMetatypeRegistry
 from nncf.common.hardware.opset import HWConfigOpName
-from nncf.torch.dynamic_graph.trace_functions import CustomTraceFunction
-from nncf.torch.dynamic_graph.trace_functions import ForwardTraceOnly
 
 ModuleAttributes = TypeVar('ModuleAttributes', bound=BaseLayerAttributes)
 
 PT_OPERATOR_METATYPES = OperatorMetatypeRegistry("operator_metatypes")
 
 
+class NamespaceTarget(Enum):
+    TORCH_NN_FUNCTIONAL = 'torch.nn.functional'
+    TORCH_TENSOR = 'torch.tensor'
+    TORCH = 'torch'
+    EXTERNAL = 'external_function'
+
+
 class PTOperatorMetatype(OperatorMetatype):
     """
     Base class for grouping PyTorch operators based on their semantic meaning.
     Each derived class represents a single semantic group - for example, AddMetatype would
     group together '__iadd__', '__add__' and '__radd__' operations which all define nodewise
     tensor addition.
-    Derived classes also specify which PyTorch functions in which modules should be patched
-    and in what manner, so that the entire group of operations is visible in the internal graph
-    representation. Grouping also allows efficient application of HW specifics to compression of
+    Also, specify from what python module corresponding operations are came from.
+    This information helps to differ functions with the same names but with different meanings.
```

Comment:
I'd rather you kept my wording - the only thing to be corrected is that `in what manner` should be removed.

Reply:
Done